src/components/svgs/stations/chongqingrt2021-basic.tsx

```diff
@@ -0,0 +1,297 @@
+                height={17}
+                fill="white"
+                fillOpacity="0"
+                stroke="#231815"
+                strokeMiterlimit="22.9"
+                strokeWidth="0.232"
+                strokeOpacity="0"
+                onPointerDown={onPointerDown}
+                onPointerMove={onPointerMove}
+                onPointerUp={onPointerUp}
+                style={{ cursor: 'move' }}
+            />
+            <g transform={`translate(${textX}, ${textY})`} textAnchor={textAnchor}>
+                <MultilineText
+                    text={names[0].split('\n')}
+                    fontSize={LINE_HEIGHT.zh}
+                    lineHeight={LINE_HEIGHT.zh}
+                    grow="up"
+                    className="rmp-name__zh"
+                    baseOffset={1}
+                />
+                <MultilineText
+                    text={names[1].split('\n')}
+                    fontSize={LINE_HEIGHT.en}
+                    lineHeight={LINE_HEIGHT.en}
+                    grow="down"
+                    className="rmp-name__en"
+                    baseOffset={1}
+                />
+                {!open && (
+                    <text
+                        dy={names[1].split('\n').length * LINE_HEIGHT.en + 2}
+                        fontSize={LINE_HEIGHT.en}
+                        dominantBaseline="hanging"
+                        className="rmp-name__zh"
+                    >
+                        (暂缓开通)
+                    </text>
+                )}
+            </g>
+        </g>
+    );
+};
+
+/**
+ * ChongqingRT2021BasicStation specific props.
+ */
+export interface ChongqingRT2021BasicStationAttributes extends StationAttributes, AttributesWithColor {
+    nameOffsetX: NameOffsetX;
+    nameOffsetY: NameOffsetY;
+    lineCode: string;
+    stationCode: string;
+    open: boolean;
+}
+
+const defaultChongqingRT2021BasicStationAttributes: ChongqingRT2021BasicStationAttributes = {
+    ...defaultStationAttributes,
+    color: [CityCode.Chongqing, 'cq1', '#e4002b', MonoColour.white],
+    nameOffsetX: 'right',
+    nameOffsetY: 'top',
+    lineCode: '1',
+    stationCode: '1',
+    open: true,
+};
+
+const ChongqingRT2021BasicAttrsComponent = (props: AttrsProps<ChongqingRT2021BasicStationAttributes>) => {
+    const { id, attrs, handleAttrsUpdate } = props;
+    const { t } = useTranslation();
+    const fields: RmgFieldsField[] = [
+        {
+            type: 'textarea',
+            label: t('panel.details.stations.common.nameZh'),
+            value: (attrs ?? defaultChongqingRT2021BasicStationAttributes).names[0],
+            onChange: val => {
+                attrs.names[0] = val.toString();
+                handleAttrsUpdate(id, attrs);
+            },
+            minW: 'full',
+        },
+        {
+            type: 'textarea',
+            label: t('panel.details.stations.common.nameEn'),
+            value: (attrs ?? defaultChongqingRT2021BasicStationAttributes).names[1],
```

Comment:
`names` could contain only one element when switching from other stations. Better be `attrs.names.at(1) ?? defaultChongqingRT2021BasicStationAttributes.names[1]`.